src/components/newProposal/steps/Confirmation.tsx

```diff
@@ -274,8 +298,50 @@ export const Confirmation = () => {
           )}
         </MainCard>
       </div>
-      <ErrorWrapper name="submit" error={errors?.root?.step4error as any}>
-        <StepNavigator isSubmitting={isSubmitting} />
+      <ErrorWrapper
+        className="flex gap-x-2 flex-row items-center"
+        name="submit"
+        error={errors?.root?.step4error as any}
+      >
+        <StepNavigator
+          isSubmitting={isSubmitting}
+          nextStepConditions={[
+            {
+              when: !isConnected,
+              content: <ConnectWalletWarning action="to submit" />,
+            },
+            {
+              when: votingPowerLoading || proposalCreationCostLoading,
+              content: <Loading className="w-5 h-5" />,
+            },
+            {
+              when: proposalCreationCostError !== null,
+              content: (
+                <Warning>
+                  Can not determine if you may create a proposal
+                </Warning>
+              ),
+            },
+            {
+              when:
+                proposalCreationCost !== null &&
+                votingPower.lt(proposalCreationCost),
+              content: <InsufficientRepWarning action="to create proposal" />,
+            },
+          ]}
+        />
+        {isConnected &&
+          proposalCreationCost !== null &&
+          votingPower.gte(proposalCreationCost) && (
+            <p>
+              Creating a proposal costs{' '}
```

Comment:
I would personally prefer "You will pay X SECOREP"
But I leave it up to you if you actually choose to replace this with it